oak_runtime/src/channel.rs

```diff
@@ -76,14 +76,22 @@ pub struct Channel {
 
     /// The name for the channel.
     ///
-    /// The name does not have to be unique and can be empty. It is used in logs to help with
-    /// identifying channels during debugging.
-    pub name: String,
+    /// The name does not have to be unique and can be empty. In logs it is
+    /// combined with the id to form a unique debug_ig to identify channels.
+    name: String,
 
     /// Weak reference to the Runtime used for sending introspection events.
     runtime_weak: Weak<Runtime>,
 }
 
+impl Channel {
+    /// Returns a unique debug_id used to identify the node in the debug output,
+    /// consisting out of the provided [`ChannelId`], and the node's name.
+    fn get_debug_id(&self, channel_id: ChannelId) -> String {
+        format!("{}({})", &self.name, channel_id)
```

Comment:
Just curious... why is this in a separate `impl Channel` block than the one below with the `new` method?

Reply:
Overlooked the other implementation block, will move it there :)